src/blockchains/cosmos.ts

```diff
@@ -0,0 +1,82 @@
+import { BlockchainHandler, BlockchainHandlerOpts } from '../blockchain-handler';
+import { AccountID } from 'caip';
+import { getConsentMessage, LinkProof } from '../utils';
+import { signTx, Tx, SignMeta, verifyTx } from '@tendermint/sig';
+const { base64ToBytes, bytesToBase64, toCanonicalJSONBytes } = require('@tendermint/belt');
```

Comment:
Please use the `uint8array` package for base64 etc.

Reply:
Not sure what `toCanonicalJSONBytes` does, but maybe we can replace it as well?

Reply:
Used uint8array for string encoding and removed tendermint/belt. CanonicalJson is meant to normalise JSON to prevent minor changes from breaking the signature. Now that I think about it, its redundant since the JSON is formed within the code itself. Replaces with JSON.stringify()

Reply:
Thanks!